src/Microsoft.Health.Dicom.Api.UnitTests/Features/Exceptions/ExceptionHandlingMiddlewareTests.cs

```diff
@@ -123,6 +123,30 @@ public async Task WhenExecutingExceptionMiddleware_GivenAnHttpContextWithNoExcep
         Assert.Equal(0, _context.Response.Body.Length);
     }
 
+    [Fact]
+    public async Task GivenAnAggregateExceptionHasTaskCanceled_WhenMiddlewareIsExecuted_ThenMessageShouldBeOverwritten()
+    {
+        var innerExceptions = new List<Exception>
+    {
```

Comment:
Is this indent correct?

Reply:
ah no. I merged too soon. Will fix in my next commit